data/json/items/ammo.json

```
@@ -788,7 +788,7 @@
     "category": "spare_parts",
     "price": 1000,
     "price_postapoc": 250,
-    "name": { "str": "copper" },
+    "name": { "str_sp": "copper" },
     "symbol": "=",
     "color": "yellow",
     "description": "Copper scraps.  Could be used to craft something, for example makeshift shotgun shells.",
```

```suggestion
    "description": "Copper bits.  Could be used to craft something, for example makeshift shotgun shells.",
```
It takes 25 of these to make a "copper scrap", by requirements/materials.json.